composer.json

```diff
@@ -28,20 +28,22 @@
 
     "autoload": {
         "psr-4": {
-            "Graze\\Supervisor\\": "src"
+            "Graze\\Supervisor\\": "src",
+            "Graze\\Supervisor\\Test\\": "tests/src"
```

Comment:
Should the test namespace go in the `autoload-dev` section?

Reply:
Yup!